src/main/java/com/amihaiemil/docker/Volume.java

```diff
@@ -50,4 +49,14 @@
      */
     JsonObject inspect() throws IOException, UnexpectedResponseException;
 
+    /**
+     * Remove a volume.
+     * @param force If true delete volumes that are in use
+     * @throws IOException If something goes wrong.
+     * @throws UnexpectedResponseException If the status response is not
+     *  the expected one (200 OK).
+     * @see <a href="https://docs.docker.com/engine/api/v1.35/#operation/VolumeDelete">Remove a volume</a>
+     */
+    void delete(Boolean force) throws IOException, UnexpectedResponseException;
```

Comment:
@bkuzmic The discussion about checked exceptions goes on and on. I believe that we should not declare them in interfaces, because they lead us to a lot of design problems. I'd like to know your opinion, and @amihaiemil 's opinion too on this subject.

Reply:
@paulodamaso I understand the problem of checked execeptions, especially when there is a proprietary business exception (for instance, other docker clients are throwing checked ``DockerException`` -- wtf is that?).

However, in this case, ``IOException`` is pretty clear and natural: you are working with an API on the network so you should expect and know what to do if there is an ``I/O`` problem. And ``UnexpectedResponseException`` is runtime, no need for the user to handle it.

Reply:
@paulodamaso @amihaiemil I agree that IOException should be declared for the reasons that @amihaiemil mentioned in his comment. I also think that it is a good practice to declare runtime exceptions in interface, just to let the user be aware of them.

Reply:
@bkuzmic Declaring exceptions here assumes about our interfaces implementations, and it should not happen. Please take a look at https://pragmaticobjects.com/chapters/001_checked_exceptions.html for some insights about this discussion. Anyways, if @amihaiemil does not see any problem declaring these exceptions, then I'm good with it.

Reply:
@paulodamaso I read that article and I understand the idea. But I think it's valid only in the cases when the interfaces can also be implemented in a non ``I/O`` manner. E.g. that ``Fraction`` from the article will almost always have an in-memory implementation. On the other hand, since this is an API client, it's obvious that most implementations will use the network behind somehow :)